### PR TITLE
test(appconnect): Add simple CLI for dev work

### PR DESCRIPTION
**Pull request by flub**
Simple CLI script for dev work on app store connect.

### CODE REVIEW

File: scripts/appconnect_cli.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+"""
+import sys
+from pprint import pprint
+
+from sentry.lang.native.appconnect import SYMBOL_SOURCES_PROP_NAME
+from sentry.models import AppConnectBuild, Project
+from sentry.tasks import app_store_connect
+from sentry.utils import json
+
+PROJECT_ID = 2
```
Comment (Swatinem) [inline scripts/appconnect_cli.py:15]: this should probably be a CLI param instead.

**Review by Swatinem — APPROVED** [review]
rubberstamping, though making the project ID mandatory on the CLI would be a good idea

**Review by relaxolotl — APPROVED** [review]
seconding the rubber stamping i'm afraid, but i have found this to be very helpful while testing my app(store?)connect changes so far.

i don't see any obvious footguns or glaring issues with the diff, although brief one-liner descriptions of each arg may be useful.

**Comment by flub** [comment]
Nice commit message:
> Add extremely pool argument parsing

s/pool/poor/

**Comment by flub** [comment]
Hi @getsentry/owners-sentry-dev, could we get an approval for you as well?

File: scripts/appconnect_cli.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+
+def main(argv):
+    if argv[0] == "dump-cfg":
```
Comment (armenzg) [inline scripts/appconnect_cli.py:22]: You're welcome to land, however, this script could be better written with [argparse](https://docs.python.org/3.6/library/argparse.html) or [click](https://pypi.org/project/click/).

**Review by armenzg — APPROVED** [review]

File: scripts/appconnect_cli.py (hunk 2 above)
Comment (flub) [reply scripts/appconnect_cli.py:22]: yeah, I know 😄 for now this is not a general purpose script for operators though so i thought full on arg parsing was a bit overkill.  thanks for approving though.

**Pull request merged**